warehouse/accounts/services.py

```diff
@@ -246,9 +248,16 @@ def __eq__(self, other):
 
 @implementer(IPasswordBreachedService)
 class HaveIBeenPwnedPasswordBreachedService:
-    def __init__(self, *, session, api_base="https://api.pwnedpasswords.com"):
+    def __init__(
+        self, *, session, api_base="https://api.pwnedpasswords.com", metrics=None
+    ):
         self._http = session
         self._api_base = api_base
+        self._metrics = metrics
+
+    def _metrics_increment(self, *args, **kwargs):
```

Comment:
I think is clearly a smell that I should get to work refactoring datadog into a proper interface/service.

Reply:
Yea, I thought of that as I was writing this too :) A nice ``IMetricsService`` with an inmemory provider could be cool. Might even make it possible to get rid of the ``notdatadog`` script, since the in memory one (or maybe "local" is a better name) could just print out directly instead of sending to ``notdatadog``.